evm/src/utils.rs

```diff
@@ -136,6 +137,18 @@ pub fn halt_to_instruction_result(
     }
 }
 
+/// Converts an `EvmVersion` into a `SpecId`
+pub fn evm_spec(evm: &EvmVersion) -> SpecId {
+    match evm {
+        EvmVersion::Istanbul => SpecId::ISTANBUL,
+        EvmVersion::Berlin => SpecId::BERLIN,
+        EvmVersion::London => SpecId::LONDON,
+        EvmVersion::Paris => SpecId::MERGE,
+        EvmVersion::Shanghai => SpecId::SHANGHAI,
+        _ => panic!("Unsupported EVM version"),
+    }
+}
```

Comment:
I know you added just Paris and Shanghai ([current](https://github.com/foundry-rs/foundry/blob/0466bb455422efcee9317480d951c8041a9f6713/cli/src/utils.rs#L88C1-L95) is just istanbul, berlin, london), but is there any reason we didn't have more versions mapped in the first place? This panics in some cases: #4873

[EvmVersion docs](https://docs.rs/ethers/latest/ethers/solc/enum.EvmVersion.html), [SpecId docs](https://docs.rs/revm-primitives/latest/revm_primitives/specification/enum.SpecId.html)

Introduced in #918: cc @onbjerg @mattsse

Reply:
I am actually not sure and is a pending question I have myself—I wanna try and document how we're using the hard forks and if there's any reason not to include them all (and if we want to, ideally in another PR).

Reply:
Let's discuss in separate issue - i dont recall a particular reason